### ISSUE TITLE
CP-ONLY Mausoleum bug returns

### ISSUE THREAD

**Issue by Tesla101a** [LoneGazebo/Community-Patch-DLL#10594]
_1. Mod Version (X.Y.Z). Current Version: 4.4.2_

4.4.2

_2. DirectX Version_

DX11

_3. List of Other Mods_

In game editor

_4. Describe the Issue_

I noticed issue #9946 is back with this latest version of the community patch.  It grants more gold than it should when expending GP, I think it may be full Vox Populi leaking into CP again.

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_

**Comment by Nu-Illasera** [comment]
NO! its my favorite "Bug", it makes me OP as hell, keep it as intentional.
